.github/workflows/CI_build.yml

```diff
@@ -0,0 +1,25 @@
+name: Continuous Integration
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: windows-2022
+    strategy:
+      fail-fast: false
+      matrix:
+        vc_arch: [32, 64]
+        vc_ver: [14.3]
+
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v3
+
+    - name: Add msbuild to PATH
+      uses: microsoft/setup-msbuild@v1
+
+    - name: run python build script
+      working-directory: .
+      run: python ./run.py --type=release --vc-arch ${{ matrix.vc_arch }} --vc-ver ${{ matrix.vc_ver }}
```

Comment:
Why did you set `--type=release`, if we're going to run on a regular basis, wouldn't we want to use `--type=master-snapshot`?

Reply:
Depends on your needs. It is also possible to have different CI builds. One on pulls,PRs with the specified type within run and another one maybe on a daily base of the master-snapshot.